### PR TITLE
Make CTFontDescriptor Send and Sync and expose attributes()

### PR DESCRIPTION
**Pull request by jrmuizel**

### CODE REVIEW

File: core-text/src/font_descriptor.rs [hunk 1]
```diff
@@ -194,6 +194,10 @@ declare_TCFType! {
 impl_TCFType!(CTFontDescriptor, CTFontDescriptorRef, CTFontDescriptorGetTypeID);
 impl_CFTypeDescription!(CTFontDescriptor);
 
+// "Font objects (CTFont, CTFontDescriptor, and associated objects) can be used
+//  simultaneously by multiple operations, work queues, or threads."
```
Comment (emilio) [inline core-text/src/font_descriptor.rs:198]: Can you link to the docs?

**Review by emilio — APPROVED** [review]

File: core-text/src/font_descriptor.rs (hunk 1 above)
Comment (jrmuizel) [reply core-text/src/font_descriptor.rs:198]: Oh I missed this comment before I r+ it...

**Pull request merged**